### DISCUSSION TITLE
How to push the converted repo to my Github entprise site after git svn clone operation.

### DISCUSSION THREAD

**Discussion by liangming2003** [community/community#32627] [Enterprise]
I was trying to migrate my SVN code to Github Enterprise repository.

I was following the following doc. 
https://docs.microsoft.com/en-us/azure/devops/repos/git/perform-migration-from-svn-to-git?view=azure-devops

everything went well and I successfully "push repository to a bare git repository" by running following commands.

```
cd c:\mytempdir git remote add bare c:\new-bare.git
git config remote.bare.push refs/remotes/*:refs/heads/* 
git push bare
```

What to do next?
where can I see the this bare git repo? 
I did not see it at my Github enterprise site [github.azc.ext.xx.com/liang-ming](https://github.azc.ext.xx.com/liang-ming) as new repo.

I was expecting the repo under c:\mytempdir was commit to a new repo at my Github enterprise site. How to implement this? thanks.

**Comment by airtower-luna** [comment]
You pushed to a bare repository on your disk. That works, but has nothing to do with any other server. A remote Git repository can be anywhere, including on your local disk. :wink: 

If you want to push your data to a repository on your GitHub enterprise server you need to create the repository there, add it as a remote (with the correct URL), and push to that remote.

See also: [Git Basics - Working with Remotes](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes)

**Reply by liangming2003** [reply]
Thanks, This time I created a repo "test-repo" on Github enterprise site and run following command to push the local repo.

cd c:\mytempdir
git remote add origin https://PAT@github.azc.ext.xx.com/liang-ming/test-repo
git push --set-upstream origin master

It failed with the error about some sdf  files larger than 100M,

```
remote: error: Please visit https://www.yammer.com/xx.com/#/threads/inGroup?type=in_group&feedId=6603278
remote: error: File HH.sdf is 104.75 MB; this exceeds GitHub Enterprise's file size limit of 100.00 MB
remote: error: File HHx.sdf is 113.50 MB; this exceeds GitHub Enterprise's file size limit of 100.00 MB
remote: error: File HH/xxx.sdf is 117.44 MB; this exceeds GitHub Enterprise's file size limit of 100.00 MB
remote: error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.
To https://github.azc.ext.xx.com/liang-ming/test-repo
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'https://PAT@github.azc.ext.xx.com/liang-ming/test-repo'
```


I searched the sdf files in c:\mytempdir, did not find any sdf file larger than 100M, so I guess it was the large file in the SVN commit history.

So I tried the BFG, it said it did not find any file larger than 100M. 
https://rtyley.github.io/bfg-repo-cleaner/,
java -jar bfg-1.14.0.jar --strip-blobs-bigger-than 100M C:\mytempdir

I also tried the LFS to track "*.sdf" files mentioned in the following link, buy it did not resolve the error.
https://github.com/git-lfs/git-lfs/wiki/Tutorial

Any solution to solve this problem? Thanks.

**Reply by airtower-luna** [reply]
I don't know your repo, so I can't comment in detail. What I notice is that [BFG won't touch certain files by default](https://rtyley.github.io/bfg-repo-cleaner/#protected-commits), you might want to check if that prevents it from removing those big files.
